config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml

```diff
@@ -0,0 +1,332 @@
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-vsphere
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-vsphere
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/cloud-provider-vsphere/ci:a74fdada
+      command:
+      - "make"
+      args:
+      - "conformance-test"
+    env:
+    - name: K8S_VERSION
+      value: release/stable-1.13.txt
+    securityContext:
+      privileged: true
+- name: ci-cloud-provider-vsphere-conformance-stable-1-12
+  interval: 12h
+  decorate: true
+  labels:
+    preset-cloud-provider-vsphere-e2e-config: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-vsphere
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-vsphere
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/cloud-provider-vsphere/ci:a74fdada
+      command:
+      - "make"
+      args:
+      - "conformance-test"
+    env:
+    - name: K8S_VERSION
+      value: release/stable-1.12.txt
+    securityContext:
+      privileged: true
+- name: ci-cloud-provider-vsphere-conformance-stable-1-11
+  interval: 12h
+  decorate: true
+  labels:
+    preset-cloud-provider-vsphere-e2e-config: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-vsphere
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-vsphere
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/cloud-provider-vsphere/ci:a74fdada
+      command:
+      - "make"
+      args:
+      - "conformance-test"
+    env:
+    - name: K8S_VERSION
+      value: release/stable-1.11.txt
+    securityContext:
+      privileged: true
+
+# Runs the e2e conformance suite against a cluster turned up with the
+# in-tree vSphere cloud provider. This job is duplicated for multiple versions
+# of Kubernetes.
+- name: ci-vsphere-conformance-latest
+  interval: 12h
+  decorate: true
+  labels:
+    preset-cloud-provider-vsphere-e2e-config: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-vsphere
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-vsphere
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/cloud-provider-vsphere/ci:a74fdada
+      command:
+      - "make"
+      args:
+      - "conformance-test"
+    env:
+    - name: CLOUD_PROVIDER
+      value: vsphere
+    - name: K8S_VERSION
+      value: ci/latest.txt
+    securityContext:
+      privileged: true
+- name: ci-vsphere-conformance-stable-1-13
+  interval: 12h
+  decorate: true
+  labels:
+    preset-cloud-provider-vsphere-e2e-config: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-vsphere
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-vsphere
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/cloud-provider-vsphere/ci:a74fdada
+      command:
+      - "make"
+      args:
+      - "conformance-test"
+    env:
+    - name: CLOUD_PROVIDER
+      value: vsphere
+    - name: K8S_VERSION
+      value: ci/latest.txt
```

Comment:
should this be `release/stable-1.13.txt`?

Reply:
Hi @neolit123,

It should. I had in fact caught it when updating for the failed bazel job. Should be good now. Thanks!